probe/docker/container.go

```diff
@@ -22,11 +22,12 @@ import (
 
 // These constants are keys used in node metadata
 const (
-	ContainerName    = "docker_container_name"
-	ContainerCommand = "docker_container_command"
-	ContainerPorts   = "docker_container_ports"
-	ContainerCreated = "docker_container_created"
-	ContainerIPs     = "docker_container_ips"
+	ContainerName        = "docker_container_name"
+	ContainerCommand     = "docker_container_command"
+	ContainerPorts       = "docker_container_ports"
+	ContainerCreated     = "docker_container_created"
+	ContainerIPs         = "docker_container_ips"
+	ContainerLabelPrefix = "docker_container_label_"
```

Comment:
Oh, thanks, that shouldn't even be there anymore, there's a new `label.go` file for this.